doc/design/oss_backup.md

```diff
@@ -0,0 +1,42 @@
+# Backups using Alibaba Cloud Object Storage Service (OSS)
+
+## Cluster configured with OSS backup
+
+To create a backup in OSS, set `backup.storageType` to `"OSS"`, supply the path (in the format "<oss-bucket-name>/<path-to-backup-object>") in `oss.path` and provide the Kubernetes secret storing the Alibaba Cloud account credentials to `oss.ossSecret`.  The secret must exist prior to backup creation. Etcd backup operator will create the bucket and object if not found. The field `oss.endpoint` means the target OSS service endpoint where backs up the data to, by default, "http://oss-cn-hangzhou.aliyuncs.com" will be used, if you want to back up the data to other regions, please follow [here](https://www.alibabacloud.com/help/doc-detail/31837.htm).
```

Comment:
Some rephrasing and formatting nits:
- Change formatting for `<oss-bucket-name>/<path-to-backup-object>` to show angle brackets `<>` correctly on Github flavored markdown.
- Grammar fix: 
    - ```The field `oss.endpoint` means the target OSS service endpoint where backs up the data to, by default...``` => ```The field `oss.endpoint` is the target OSS service endpoint where the data is backed up. By default...```
- `please follow [here]` ==> `please specify another endpoint from [the list of region endpoints]`
```suggestion
To create a backup in OSS, set `backup.storageType` to `"OSS"`, supply the path (in the format `<oss-bucket-name>/<path-to-backup-object>`) in `oss.path` and provide the Kubernetes secret storing the Alibaba Cloud account credentials to `oss.ossSecret`.  The secret must exist prior to backup creation. Etcd backup operator will create the bucket and object if not found. The field `oss.endpoint` is the target OSS service endpoint where the data is backed up. By default, `http://oss-cn-hangzhou.aliyuncs.com` will be used. If you want to back up the data to other regions, please specify another endpoint from [the list of region endpoints](https://www.alibabacloud.com/help/doc-detail/31837.htm).
```